<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>697</width>
    <height>490</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Pixel::Booster</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources/icons/icons.qrc">
    <normaloff>:/app_icon/app_icon.svg</normaloff>:/app_icon/app_icon.svg</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QMdiArea" name="image_mdi_area_"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>697</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionOpen_Recent"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
    <addaction name="actionSave_All"/>
    <addaction name="actionClose"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="separator"/>
    <addaction name="actionCut"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="actionDelete"/>
    <addaction name="actionSelect_All"/>
    <addaction name="separator"/>
    <addaction name="actionAdd_Text"/>
    <addaction name="actionCreate_Shadow"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionTile_Size"/>
    <addaction name="actionShow_Grid"/>
    <addaction name="actionShow_Divisions"/>
   </widget>
   <widget class="QMenu" name="menuLayers">
    <property name="title">
     <string>Layers</string>
    </property>
    <addaction name="actionImage_Size"/>
    <addaction name="actionClear"/>
    <addaction name="actionClear_All"/>
   </widget>
   <widget class="QMenu" name="menuPalette">
    <property name="title">
     <string>Palette</string>
    </property>
    <addaction name="actionTransparency"/>
    <addaction name="separator"/>
    <addaction name="actionLoad_Palette"/>
    <addaction name="actionSave_Palette"/>
    <addaction name="actionEdit_Palette"/>
    <addaction name="separator"/>
    <addaction name="actionClear_Mask"/>
    <addaction name="actionInvert_Mask"/>
    <addaction name="actionAuto_Mask"/>
   </widget>
   <widget class="QMenu" name="menuWindow">
    <property name="title">
     <string>Window</string>
    </property>
    <widget class="QMenu" name="menuSet_Language">
     <property name="title">
      <string>Set Language</string>
     </property>
     <addaction name="actionEN_US"/>
     <addaction name="actionPT_BR"/>
    </widget>
    <addaction name="menuSet_Language"/>
    <addaction name="separator"/>
    <addaction name="actionCascade"/>
    <addaction name="actionTile_Subwindows"/>
    <addaction name="separator"/>
    <addaction name="actionNormal_Size"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionManual"/>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuLayers"/>
   <addaction name="menuPalette"/>
   <addaction name="menuWindow"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="action_toolBar">
   <property name="windowTitle">
    <string>Action Bar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="separator"/>
   <addaction name="actionUndo"/>
   <addaction name="actionRedo"/>
   <addaction name="separator"/>
   <addaction name="actionCut"/>
   <addaction name="actionCopy"/>
   <addaction name="actionPaste"/>
   <addaction name="separator"/>
   <addaction name="actionAdd_Text"/>
   <addaction name="separator"/>
   <addaction name="actionTile_Size"/>
   <addaction name="actionShow_Grid"/>
   <addaction name="actionShow_Divisions"/>
   <addaction name="separator"/>
   <addaction name="actionImage_Size"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="drawing_toolBar">
   <property name="windowTitle">
    <string>Tool Bar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSelection_Tool"/>
   <addaction name="actionZoom_Tool"/>
   <addaction name="actionFill_Tool"/>
   <addaction name="actionPencil_Tool"/>
   <addaction name="actionLine_Tool"/>
   <addaction name="actionRectangle_Tool"/>
   <addaction name="actionEllipse_Tool"/>
   <addaction name="separator"/>
   <addaction name="actionTransparency"/>
  </widget>
  <widget class="QDockWidget" name="dockWidget">
   <property name="features">
    <set>QDockWidget::NoDockWidgetFeatures</set>
   </property>
   <property name="windowTitle">
    <string>Edit Area</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QScrollArea" name="scrollArea">
       <property name="styleSheet">
        <string notr="true">background-image: url(:/images/lines_background_color.png);</string>
       </property>
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>100</width>
          <height>207</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <property name="spacing">
          <number>0</number>
         </property>
         <item row="1" column="0">
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="2">
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="1">
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="1">
          <widget class="QFrame" name="frame">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true">background-image: url(:/images/transparent_background.png);</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="ImageEditWidget" name="edit_widget" native="true">
              <property name="maximumSize">
               <size>
                <width>1</width>
                <height>1</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="2" column="1">
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="zoom_horizontalSlider">
       <property name="maximum">
        <number>6</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="tickPosition">
        <enum>QSlider::TicksBelow</enum>
       </property>
       <property name="tickInterval">
        <number>1</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidget_2">
   <property name="features">
    <set>QDockWidget::NoDockWidgetFeatures</set>
   </property>
   <property name="windowTitle">
    <string>Color Palette</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="color_main_pushButton">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="color_alt_pushButton">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QWidget" name="widget" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionNew">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources/icons/icons.qrc">
     <normaloff>:/icons/new_file.png</normaloff>:/icons/new_file.png</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources/icons/icons.qrc">
     <normaloff>:/icons/open_file.png</normaloff>:/icons/open_file.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionOpen_Recent">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Open Recent</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources/icons/icons.qrc">
     <normaloff>:/icons/save_file.png</normaloff>:/icons/save_file.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources/icons/icons.qrc">
     <normaloff>:/icons/save_file_as.png</normaloff>:/icons/save_file_as.png</iconset>
   </property>
   <property name="text">
    <string>Save As</string>
   </property>
  </action>
  <action name="actionSave_All">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources/icons/icons.qrc">
     <normaloff>:/icons/save_all_file.png</normaloff>:/icons/save_all_file.png</iconset>
   </property>
   <property name="text">
    <string>Save All</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources/icons/icons.qrc">
     <normaloff>:/icons/undo.png</normaloff>:/icons/undo.png</iconset>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources/icons/icons.qrc">
     <normaloff>:/icons/redo.png</normaloff>:/icons/redo.png</iconset>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources/icons/icons.qrc">
     <normaloff>:/icons/cut.png</normaloff>:/icons/cut.png</iconset>
   </property>
   <property name="text">
    <string>Cut</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources/icons/icons.qrc">
     <normaloff>:/icons/copy.png</normaloff>:/icons/copy.png</iconset>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources/icons/icons.qrc">
     <normaloff>:/icons/paste.png</normaloff>:/icons/paste.png</iconset>
   </property>
   <property name="text">
    <string>Paste</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources/icons/icons.qrc">
     <normaloff>:/icons/delete.png</normaloff>:/icons/delete.png</iconset>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </action>
  <action name="actionSelect_All">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Select All</string>
   </property>
  </action>
  <action name="actionAdd_Text">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources/icons/icons.qrc">
     <normaloff>:/icons/text_edit.png</normaloff>:/icons/text_edit.png</iconset>
   </property>
   <property name="text">
    <string>Add Text</string>
   </property>
  </action>
  <action name="actionCreate_Shadow">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Create Shadow</string>
   </property>
  </action>
  <action name="actionTile_Size">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources/icons/icons.qrc">
     <normaloff>:/icons/tile_size.png</normaloff>:/icons/tile_size.png</iconset>
   </property>
   <property name="text">
    <string>Tile Size</string>
   </property>
  </action>
  <action name="actionShow_Grid">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources/icons/icons.qrc">
     <normaloff>:/icons/show_grid.png</normaloff>:/icons/show_grid.png</iconset>
   </property>
   <property name="text">
    <string>Show Grid</string>
   </property>
  </action>
  <action name="actionShow_Divisions">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources/icons/icons.qrc">
     <normaloff>:/icons/show_divisions.png</normaloff>:/icons/show_divisions.png</iconset>
   </property>
   <property name="text">
    <string>Show Divisions</string>
   </property>
  </action>
  <action name="actionImage_Size">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources/icons/icons.qrc">
     <normaloff>:/icons/resize.png</normaloff>:/icons/resize.png</iconset>
   </property>
   <property name="text">
    <string>Image Size</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </action>
  <action name="actionClear_All">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Clear All</string>
   </property>
  </action>
  <action name="actionTransparency">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources/icons/icons.qrc">
     <normaloff>:/icons/transparency.png</normaloff>:/icons/transparency.png</iconset>
   </property>
   <property name="text">
    <string>Transparency</string>
   </property>
  </action>
  <action name="actionLoad_Palette">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Load Palette</string>
   </property>
  </action>
  <action name="actionSave_Palette">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Save Palette</string>
   </property>
  </action>
  <action name="actionEdit_Palette">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Edit Palette</string>
   </property>
  </action>
  <action name="actionClear_Mask">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Clear Mask</string>
   </property>
  </action>
  <action name="actionInvert_Mask">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Invert Mask</string>
   </property>
  </action>
  <action name="actionAuto_Mask">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Auto Mask</string>
   </property>
  </action>
  <action name="actionPT_BR">
   <property name="icon">
    <iconset resource="../resources/icons/icons.qrc">
     <normaloff>:/language_icons/flag_pt_br.png</normaloff>:/language_icons/flag_pt_br.png</iconset>
   </property>
   <property name="text">
    <string>PT_BR</string>
   </property>
  </action>
  <action name="actionEN_US">
   <property name="icon">
    <iconset resource="../resources/icons/icons.qrc">
     <normaloff>:/language_icons/flag_en_us.png</normaloff>:/language_icons/flag_en_us.png</iconset>
   </property>
   <property name="text">
    <string>EN_US</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="../resources/icons/icons.qrc">
     <normaloff>:/app_icon/app_icon.svg</normaloff>:/app_icon/app_icon.svg</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionManual">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Instructions</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="actionSelection_Tool">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources/icons/icons.qrc">
     <normaloff>:/icons/selection_tool.png</normaloff>:/icons/selection_tool.png</iconset>
   </property>
   <property name="text">
    <string>Selection Tool</string>
   </property>
  </action>
  <action name="actionZoom_Tool">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources/icons/icons.qrc">
     <normaloff>:/icons/zoom_tool.png</normaloff>:/icons/zoom_tool.png</iconset>
   </property>
   <property name="text">
    <string>Zoom Tool</string>
   </property>
  </action>
  <action name="actionFill_Tool">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources/icons/icons.qrc">
     <normaloff>:/icons/fill_tool.png</normaloff>:/icons/fill_tool.png</iconset>
   </property>
   <property name="text">
    <string>Fill Tool</string>
   </property>
  </action>
  <action name="actionPencil_Tool">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources/icons/icons.qrc">
     <normaloff>:/icons/pencil_tool.png</normaloff>:/icons/pencil_tool.png</iconset>
   </property>
   <property name="text">
    <string>Pencil Tool</string>
   </property>
  </action>
  <action name="actionLine_Tool">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources/icons/icons.qrc">
     <normaloff>:/icons/line_tool.png</normaloff>:/icons/line_tool.png</iconset>
   </property>
   <property name="text">
    <string>Line Tool</string>
   </property>
  </action>
  <action name="actionRectangle_Tool">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources/icons/icons.qrc">
     <normaloff>:/icons/rectangle_tool.png</normaloff>:/icons/rectangle_tool.png</iconset>
   </property>
   <property name="text">
    <string>Rectangle Tool</string>
   </property>
  </action>
  <action name="actionEllipse_Tool">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources/icons/icons.qrc">
     <normaloff>:/icons/ellipse_tool.png</normaloff>:/icons/ellipse_tool.png</iconset>
   </property>
   <property name="text">
    <string>Ellipse Tool</string>
   </property>
  </action>
  <action name="actionCascade">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources/icons/icons.qrc">
     <normaloff>:/icons/cascade.png</normaloff>:/icons/cascade.png</iconset>
   </property>
   <property name="text">
    <string>Cascade</string>
   </property>
  </action>
  <action name="actionNormal_Size">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Normal Size</string>
   </property>
  </action>
  <action name="actionTile_Subwindows">
   <property name="icon">
    <iconset resource="../resources/icons/icons.qrc">
     <normaloff>:/icons/tile_windows.png</normaloff>:/icons/tile_windows.png</iconset>
   </property>
   <property name="text">
    <string>Tile Subwindows</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ImageEditWidget</class>
   <extends>QWidget</extends>
   <header>widgets/image_edit_widget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resources/icons/icons.qrc"/>
 </resources>
 <connections/>
</ui>
